crates/pathfinder/src/rpc/v02/method/get_transaction_by_hash.rs

```diff
@@ -0,0 +1,176 @@
+use anyhow::Context;
+
+use crate::core::StarknetTransactionHash;
+use crate::rpc::v02::types::reply::Transaction;
+use crate::rpc::v02::RpcContext;
+use crate::storage::StarknetTransactionsTable;
+
+#[derive(serde::Deserialize, Debug, PartialEq, Eq)]
+pub struct GetTransactionByHashInput {
+    transaction_hash: StarknetTransactionHash,
+}
+
+crate::rpc::error::rpc_error_subset!(GetTransactionByHashError: TxnHashNotFound);
```

Comment:
I'm now wondering if the macro name is descriptive enough.. maybe it should include some verb like `generate_`?

Reply:
Yeah, I think something including `generate` would be more descriptive here.

Reply:
Okay will PR that